spec/support/tom_select_helper.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module TomSelectHelper
+  def select_tom_select(value, from:)
+    container = find(:id, from)
+
+    within(container) do
+      find('.ts-control').send_keys(value)
+    end
+
+    all('.ts-dropdown .ts-dropdown-content .option', text: /#{Regexp.quote(value)}/i)[0].click
```

Comment:
Here `all` is potentially selecting multiple options (eg a value of "Distributor" could select "Distributor1" and "Distributor2"), then clicking on the first one. 
I think what we want is to use `find`, and I'd suggest against using Regexp if it's not needed. We would be looking for an exact match.